js/lib/beautify-html.js

```diff
@@ -524,7 +524,7 @@
                     matched = false;
 
                 this.pos = start_pos;
-                input_char = this.input.charAt(this.pos);
+                var input_char = this.input.charAt(this.pos);
```

Comment:
A question. JSLint practices usually says that declarations should be in the top of function blocks due to hoisting. It seems like this practice is upheld in this function block as well.

Should the declaration of 'input_char' be moved to the rest of the declaration chain?